### ISSUE TITLE
ARI verifying error

### ISSUE THREAD

**Issue by PawelBaran2001** [jpauwels/hartufo#23]
When using this: 
``` py
Ari(base_dir / "ARI", features_spec=HrirSpec(samplerate=config.hrir_samplerate, side='left', domain=domain), subject_ids='first', verify=True, download=True)
```
there is an error poiting that anchro.mat file is corrupted. I don't think thats true, because I can import it onto matlab, and it weights 12 KB. No matter if i download it manually from browser or through the library, the filesize is the same and verifying throws error. I think thats might be a problem with wrong checksum in checksums.py. Maybe they updated the file or something

**Comment by jpauwels** [comment]
Hi @PawelBaran2001, thanks for flagging this. 99.9% sure that means the `.mat` file was updated on the server indeed. Will verify and update the checksum. For now you can get around it by setting `verify=False`.

**Comment by jpauwels** [comment]
@PawelBaran2001, I confirmed that there is indeed a new version of `anthro.mat` on the ARI servers indeed. Somewhat unexpectedly for this old data, it does contain previously unreleased data. Part of the new data are the names of the subject, breaching confidentiality. So I'm going to check with ARI whether this new release was intentional before commiting the new checksum.

**Comment by jpauwels** [comment]
Hi @PawelBaran2001, I checked this with the people from ARI and just released version 0.6.6 which brings the checksum in line with the file on their servers. You will have to delete your existing `anthro.mat` file manually, but after you `pip install -U hartufo`, the correct version will be downloaded and all checks will pass.

I've got a breaking change planned for what will be `v0.7.0` soon(ish), so I suggest you to join the [mailing list](https://github.com/jpauwels/hartufo/blob/main/README.md). Then you'll receive a message with more info once that is released.
